base/applications/iexplore/iexplore.inf

```diff
@@ -16,10 +16,13 @@ HKCR,"CLSID\%CLSID_InternetExplorer%\LocalServer32",,,"""%16422%\Internet Explor
 HKCR,"CLSID\%CLSID_InternetExplorer%\ProgID",,,"InternetExplorer.Application.1"
 HKCR,"CLSID\%CLSID_InternetExplorer%\VersionIndependentProgID",,,"InternetExplorer.Application"
 
-HKCR,"CLSID\%CLSID_Internet%\DefaultIcon",,,"shdoclc.dll,-190"
-HKCR,"CLSID\%CLSID_Internet%\Shell",,,"OpenHomePage"
-HKCR,"CLSID\%CLSID_Internet%\Shell\OpenHomePage",,,"Open &Home Page"
-HKCR,"CLSID\%CLSID_Internet%\Shell\OpenHomePage\Command",,,"""%16422%\Internet Explorer\iexplore.exe"""
+HKCR,"CLSID\%CLSID_Internet%",,,"Internet"
+HKCR,"CLSID\%CLSID_Internet%","InfoTip",0x00020000,"@%SystemRoot%\system32\shell32.dll,-22546"
+HKCR,"CLSID\%CLSID_Internet%","LocalizedString",0x00020000,"@%SystemRoot%\system32\shell32.dll,-9222"
+HKCR,"CLSID\%CLSID_Internet%\DefaultIcon",,0x00020000,"%16422%\Internet Explorer\iexplore.exe,-1"
+HKCR,"CLSID\%CLSID_Internet%\Shell",,,"open"
+HKCR,"CLSID\%CLSID_Internet%\Shell\open",,,""
+HKCR,"CLSID\%CLSID_Internet%\Shell\open\Command",,,"""%16422%\Internet Explorer\iexplore.exe"""
```

Comment:
Maybe use an URL scheme here, so it would start default assigned browser instead. I know Windows XP/2003 opens Internet Explorer by clicking on this, but suggested behavior will be more useful.